utility/fetchClientId.js

```diff
@@ -0,0 +1,100 @@
+/**	Utility function that fetches client IDs (GA3, GA4, Segment, Facebook) from the browser
+ * and sends them asyncrhonously to the Littledata server.
+ * You should run this function when all the tracking libraries have been loaded and when you have
+ * the Shopfy cartToken or checkoutToken created by the Shopify API.
+ *
+ * @param {*} shopifyToken Shopify Cart token or Shopify Checkout token
+ * @param {*} ga3PropertyId GA3 Property ID (ex. UA-XXXX-X)
+ * @param {*} ga4MeasurementId GA4 Measurement ID (ex. G-XXXX)
+ */
+
+const fetchClientId = async ({
+	shopifyToken,
```

Comment:
rename to cartID or cartToken in order to be more explicit

Reply:
intentionally like that @lageorgem . they can be sending either cart or checkout token, depending on what do they form on the frontend and use as a cart